Content.Shared/Chat/V2/Components/CanVerbalChatComponent.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Robust.Shared.GameStates;
+
+namespace Content.Shared.Chat.V2.Components;
+
+/// <summary>
+/// Declares that this entity can chat in local chat
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class CanVerbalChatComponent : Component
```

Comment:
These are somewhat orphaned at the moment (voth CanVerbal and CanVisual). Their ranges might well be determined by YAML specification of their chat channel. For the time being I'm keeping them around; we'll need components to control access to channels to stop smart fridges wiggling their non-existant eyebrows.